[drm] Missing/Incorrect functions in DRM platform

### Issue description
Building with Software renderer and DRM platform has errors about function names.

DRM platform either has missing or incorrect named functions

> rcore_drm.c:845:26: error: ‘SW_COLOR_BUFFER_BITS’ undeclared (first use in this function); did you mean ‘SW_COLOR_BUFFER_BIT’?
>   845 |     const uint32_t bpp = SW_COLOR_BUFFER_BITS;

> rcore_drm.c:902:5: error: implicit declaration of function ‘swBlitFramebuffer’; did you mean ‘rlBlitFramebuffer’? [-Wimplicit-function-declaration]
>   902 |     swBlitFramebuffer(0, 0, width, height, 0, 0, width, height, SW_RGBA, SW_UNSIGNED_BYTE, dumbBuffer);

### Environment
Master [f36533c](https://github.com/raysan5/raylib/commit/f36533cbd8feead845a2e0b4568ec0484a15cd10)
```
Gentoo 6.19.2
KDE Plasma 6.5.5
Wayland
Software rendering
```

### Code Example

Building with these flags
```
GRAPHICS_API_OPENGL11_SOFTWARE
PLATFORM DRM
OPENGL_VERSION "Software"
```

--- a/src/platforms/rcore_desktop_sdl.c
+++ b/src/platforms/rcore_desktop_sdl.c
@@ -1528,6 +1528,10 @@ void PollInputEvents(void)
                             if ((width + borderLeft + borderRight != usableBounds.w) && (height + borderTop + borderBottom != usableBounds.h)) FLAG_CLEAR(CORE.Window.flags, FLAG_WINDOW_MAXIMIZED);
                         }
                         #endif
+
+                        #if defined(GRAPHICS_API_OPENGL_SOFTWARE)
+                        swResize(width, height);
+                        #endif
                     } break;
 
                     case SDL_WINDOWEVENT_ENTER: CORE.Input.Mouse.cursorOnScreen = true; break;

--- a/src/platforms/rcore_desktop_win32.c
+++ b/src/platforms/rcore_desktop_win32.c
@@ -2091,6 +2091,10 @@ static void HandleWindowResize(HWND hwnd, int *width, int *height)
 
     CORE.Window.screenScale = MatrixScale( (float)CORE.Window.render.width/CORE.Window.screen.width,
         (float)CORE.Window.render.height/CORE.Window.screen.height, 1.0f);
+
+    #if defined(GRAPHICS_API_OPENGL_SOFTWARE)
+    swResize(clientSize.cx, clientSize.cy);
+    #endif
 }
 
 // Update window style

--- a/src/platforms/rcore_drm.c
+++ b/src/platforms/rcore_drm.c
@@ -837,14 +837,8 @@ void SwapScreenBuffer(void)
     uint32_t height = mode->vdisplay;
 
     // Dumb buffers use a fixed format based on bpp
-#if SW_COLOR_BUFFER_BITS == 24
     const uint32_t bpp = 32;    // 32 bits per pixel (XRGB8888 format)
     const uint32_t depth = 24;  // Color depth, here only 24 bits, alpha is not used
-#else
-    // REVIEW: Not sure how it will be interpreted (RGB or RGBA?)
-    const uint32_t bpp = SW_COLOR_BUFFER_BITS;
-    const uint32_t depth = SW_COLOR_BUFFER_BITS;
-#endif
 
     // Create a dumb buffer for software rendering
     struct drm_mode_create_dumb creq = { 0 };
@@ -899,7 +893,7 @@ void SwapScreenBuffer(void)
 
     // Copy the software rendered buffer to the dumb buffer with scaling if needed
     // NOTE: RLSW will make a simple copy if the dimensions match
-    swBlitFramebuffer(0, 0, width, height, 0, 0, width, height, SW_RGBA, SW_UNSIGNED_BYTE, dumbBuffer);
+    swBlitPixels(0, 0, width, height, 0, 0, width, height, SW_RGBA, SW_UNSIGNED_BYTE, dumbBuffer);
 
     // Unmap the buffer
     munmap(dumbBuffer, creq.size);

